lib/extendedkey.ex

```diff
@@ -78,27 +82,105 @@ defmodule Bitcoinex.ExtendedKey do
       end
     end
 
-    @spec from_string(String.t()) :: {:ok, t()} | {:error, String.t()}
-    def from_string(pathstr) do
+    @spec to_bin(t()) :: {:ok, binary} | {:error, String.t()}
+    def to_bin(%__MODULE__{child_nums: child_nums}) do
       try do
-        {:ok, %__MODULE__{child_nums: tfrom_string(String.split(pathstr, "/"))}}
+        {:ok, tto_bin(child_nums, <<>>)}
```

Comment:
what's meaning of the prefix `t` for private function like `tfrom_string`, `tto_bin`?

Reply:
I named the recursive helper functions differently, but there's no real reason for that. I'll remove and make them all the same.